contrib/security/PSS/patches/ml-pipeline-ui.yaml

```diff
@@ -13,6 +13,14 @@ spec:
           seccompProfile:
             type: RuntimeDefault
           runAsNonRoot: true
+          runAsUser: 1000
+          runAsGroup: 0
           capabilities:
             drop:
             - ALL
+      initContainers:
```

Comment:
Please remove all the istio initcontainer patches from all 
deployment patches. They are injected at runtime and changed by the general istio configuration you modified with sed

Reply:
initContainers have been removed in the commit https://github.com/kubeflow/manifests/pull/2866/commits/e40311f4032ed586f8144574bb4e8c742687e5ed